build/datagen-steps/meta.js

```diff
@@ -0,0 +1,77 @@
+import fs from "fs";
+import path from "path";
+import { marked } from "marked";
+
+marked.setOptions({
+  renderer: new marked.Renderer(),
+  gfm: true,
+  tables: true,
+  breaks: false,
+  pedantic: false,
+  sanitize: false,
+  smartLists: true,
+  smartypants: false,
```

Comment:
```suggestion
  smartypants: false,
  headerIds: false,
```

Until looking at your `snap` test files I hadn't noticed these IDs were getting added to the headers, and they're totally unnecessary just create larger files than needed!

Reply:
Of course, unfortunately, after this change tests will fail until the snaps are updated, but that ought to be an easy edit.

Reply:
You got it. Snap files are a breeze to update :)

Reply:
Awesome, thx!